src/Hangfire.Core/Dashboard/Content/js/hangfire.js

```diff
@@ -187,7 +187,8 @@
         return StatisticsPoller;
     })();
 
-    hangFire.Page = (function() {
+    hangFire.Page = (function () {
```

Comment:
it might be worth moving the different sections out before it gets to large aswell. might be worth talking to rob about this